src/Entity/ItemId.php

```diff
@@ -45,7 +45,9 @@ private function assertValidIdFormat( $idSerialization ) {
 	}
 
 	/**
-	 * @return int
+	 * @see Int32EntityId::getNumericId
+	 *
+	 * @return int Guaranteed to be a unique integer in the range [1..2147483647].
```

Comment:
The `@see Int32EntityId` is critical here. I did not wanted to repeat the entirety of the interface. "Unique" means that non-equal EntityId objects are guaranteed to return different integer numbers. EntityId objects that are equal obviously return the same numeric ID.

This is not a GUID. Nothing is said about "global" or something.